### PR TITLE
Finishing links for Contributor Summit

### PR DESCRIPTION
**Pull request by qedrakmar**
Finishing links for Contributor Summit

/assign @jberkus 
/assign @pnbrown

### CODE REVIEW

**Comment by qedrakmar** [comment]
@jberkus I think we're good to merge.

File: static/_redirects [hunk 1]
```diff
@@ -39,5 +39,6 @@
 /summit           /events/2023/kcsna  302
 /summit/reg       https://cvent.me/qRPey3 302
 /summit/cfp       https://forms.gle/htQSHpot9rp1csDz8 302
-# /summit/unconf    LINK UNCONFERENCE GITHUB ISSUE HERE 302
-# /summit/notes     LINK GOOGLE DRIVE NOTES FOLDER HERE 302
+/summit/unconf    https://github.com/kubernetes/community/issues/7531 302
+/summit/notes     https://drive.google.com/drive/folders/1plMF2WG5AKcdzQwQ5PQbmFX2xhkreUis 302
```
Comment (sftim) [inline static/_redirects:43]: This is a private Google doc - is that right?

Comment (qedrakmar) [reply static/_redirects:43]: It's a folder and should be shared with the org.  Can you not access it?  If so, we can look at the permissions.

Comment (jberkus) [reply static/_redirects:43]: I can access it

Comment (sftim) [reply static/_redirects:43]: OK, so it's meant to require a login. So long as that's intended.

**Review by mrbobbytables — COMMENTED** [review]
/approve

**Pull request merged**